LICENSE

```diff
@@ -0,0 +1,407 @@
+Attribution-NonCommercial 4.0 International
```

Comment:
Do we definitely want a copy rather than just the link?

Reply:
If we add a copy then GitHub adds it to the repo metadata like this
<img width="221" alt="image" src="https://github.com/dxw/playbook/assets/8467576/8c5ecb8a-e2c3-4ecb-a7ed-d17c477b1735">
<img width="1588" alt="image" src="https://github.com/dxw/playbook/assets/8467576/985872ff-ab0d-4f2f-bd2b-786c2e18c8f4">

Reply:
Nice